apps/central-scan/backend/src/central_scanner_app.ts

```diff
@@ -612,6 +519,8 @@ export async function buildCentralScannerApp({
     }
   );
 
+  // TODO: Update end_to_end_hmpb.test.ts to use configureFromBallotPackageOnUsbDrive and then
+  // remove this, as it's no longer used by the frontend
```

Comment:
Sorry for missing these in my PR! It seems fragile that the backend e2e tests need to reconstruct the sequence of requests the frontend makes, and I'm wondering if we should deprecate these e2e tests for that reason. But the only alternative I can think of for this type of coverage is an integration test and I know we're not convinced that's the best place to invest. Ticketed in https://github.com/orgs/votingworks/projects/23/views/23?filterQuery=label%3A%22proj%3Asys-config%22&pane=issue&itemId=26607650 and I can pick this up in the next 1-2 weeks if you don't think this is urgent.

Reply:
No worries at all! It also wasn't immediately apparent to me that they were no longer used

> It seems fragile that the backend e2e tests need to reconstruct the sequence of requests the frontend makes

Yeah I agree that we should avoid these tests of longer sequences. They're fragile and harder to reason about than more atomic tests. I do, however, see value in tests of smaller sequences, like the common "POST followed by a GET" test or a "configure followed by a scan" test, where the underlying store isn't mocked and we're actually testing store logic, through the top-level API

Thank you for opening a ticket! I def don't think it's urgent